examples/Demo.Console.NET6/Demo.Console.NET6.csproj

```diff
@@ -20,6 +20,10 @@
     <PackageReference Include="Microsoft.TeamFoundationServer.Client" Version="16.153.0" />
     <PackageReference Include="NLog" Version="4.7.6" />
     <PackageReference Include="NLog.Extensions.Logging" Version="1.6.5" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
+      <PrivateAssets>all</PrivateAssets>

```

Comment:
Curious why we needed to add these? Just stylecop to trigger stylecop should have been enough.

Reply:
I dont know why, stylecop did not trigger any warnings or errors on this project.

Reply:
That's curious... seems like we should be able to follow the same pattern as our other projects. I think it would be good to understand why this is needed before we merge.